scripts/llm/ptq.py

```diff
@@ -97,16 +105,27 @@ def main():
         dtype=args.dtype,
         generate_sample=args.generate_sample,
     )
-
     quantizer = quantization.Quantizer(quantization_config, export_config)
-    model = quantization.load_with_modelopt_layer_spec(
-        args.nemo_checkpoint,
-        args.calibration_tp,
-        args.calibration_pp,
+
+    model, trainer = quantization.load_with_modelopt_layer_spec(
+        nemo_checkpoint_path=args.nemo_checkpoint,
+        tensor_model_parallel_size=args.calibration_tp,
+        pipeline_model_parallel_size=args.calibration_pp,
+        num_nodes=args.num_nodes,
         ckpt_load_strictness=args.ckpt_load_strictness,
     )
     model = quantizer.quantize(model)
-    quantizer.export(model, args.nemo_checkpoint)
+
+    if args.save_as_torch:
+        # Standard NeMo 2.0 checkpoint format
+        trainer.save_checkpoint(args.export_path)
+        if is_global_rank_zero():
+            TrainerContext.from_trainer(trainer).io_dump(
+                ckpt_to_context_subdir(args.export_path), yaml_attrs=["model"]
+            )
+    else:
+        # TRT-LLM
+        quantizer.export(model, args.nemo_checkpoint)
```

Comment:
Can you update one of the CI tests to save nemo instead of default trtllm and also check `modelopt_state` folder is present?